### DISCUSSION TITLE
Proper or elegant alternative to cmake_configuration

### DISCUSSION THREAD

**Discussion by DemingQiao** [xmake-io/xmake#3883] [Q&A] [answered]
I am a beginner in xmake, and I am converting the previous cmake project to xmake. I would like to ask how to generate
configuration files, such as 
config.h
version.h
export.h
In cmake, I use cmak_configuration to generate these configuration files,
Is there a similar way in xmake

**Comment by waruqi** [comment, marked as answer]
https://xmake.io/#/manual/project_target?id=targetadd_configfiles

**Reply by DemingQiao** [reply]
Thanks a lot
